src/context/StoreProvider/types.ts

```diff
@@ -8,49 +8,49 @@ export type StoreReducer = Reducer<State, Action>;
 export type Dispatch = (action: Action) => void;
 
 export type Page =
-  "home"
+  | "home"
   | "link"
   | "view"
   | "create"
   | "edit"
   | "add_comment"
-  | "add_story_relations"
-;
+  | "add_story_relations";
 
 export interface State {
   page?: Page;
   pageParams?: any;
   context?: TicketContext;
-  linkStorySearchResults?: { loading: boolean, list: StorySearchItem[] };
-  linkedStoriesResults?: { loading: boolean, list: StoryItem[] };
-  relationsStoriesResults?: { loading: boolean, list: StoryItem[] },
+  linkStorySearchResults?: { loading: boolean; list: StorySearchItem[] };
+  linkedStoriesResults?: { loading: boolean; list: StoryItem[] };
+  relationsStoriesResults?: { loading: boolean; list: StoryItem[] };
   dataDependencies?: any; // ToDo: need types
 
   // ...
 
-  _error?: Error|unknown;
+  _error?: Error | unknown;
 }
 
 export type Action =
-  | { type: "changePage", page: Page, params?: object }
-  | { type: "loadContext", context: Context }
+  | { type: "changePage"; page: Page; params?: object }
+  | { type: "loadContext"; context: Context }
   | { type: "linkStorySearchListLoading" }
-  | { type: "linkStorySearchList", list: StorySearchItem[] }
+  | { type: "linkStorySearchList"; list: StorySearchItem[] }
   | { type: "linkStorySearchListReset" }
-  | { type: "linkedStoriesList", list: StoryItem[] }
+  | { type: "linkedStoriesList"; list: StoryItem[] }
   | { type: "linkedStoriesListLoading" }
-  | { type: "loadDataDependencies", deps: any }
-  | { type: "loadDependencies", deps: any }
-  | { type: "relationsStoriesList", list: StoryItem[] }
+  | { type: "loadDataDependencies"; deps: any }
+  | { type: "loadDependencies"; deps: any }
+  | { type: "relationsStoriesList"; list: StoryItem[] }
   | { type: "relationsStoriesListLoading" }
```

Comment:
we don't need actions anymore, you made the data to be loaded through react-query